odoo/addons/test_lint/tests/eslintrc

```diff
@@ -29,6 +29,10 @@
             {
                 "selector": "ClassDeclaration[superClass.name='Component']:not(:has(ClassBody.body > PropertyDefinition[key.name='template'][static='true']))",
                 "message": "Component should declare static template"
+            },
+            {
+                "selector": "CallExpression[callee.name='_t'] > Literal:nth_child(1)[value=/(.*%s){2,}/]",
```

Comment:
bad selector: `nth_child` → `nth-child`. Even better: `first-child`
I don't think Literal is necessary:
```suggestion
                "selector": "CallExpression[callee.name='_t'] > :first-child[value=/(.*%s){2,}/]",
```

Reply:
apparently the part about nth_child was already fixed, but in the wrong commit. I moved it to the right place and made the other changes 🙂